src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportFileUploader.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import {
+  notification,
+  Space,
+  Typography,
+} from "antd";
+import { DragUpload } from "../../../../components/files/DragUpload";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+import { SampleMetadataImportSteps } from "./SampleMetadataImportSteps";
+
+export function SampleMetadataImportFileUploader(props) {
```

Comment:
Destructure `props` to `{projectId}`

Reply:
Please see 37e33cb8abcadf2cd1efa57175d059c701b551d6.